### PR TITLE
re-export solve_only call chain

### PR DESCRIPTION
**Pull request by healeyq3**
Addressing request stated here: https://github.com/cvxgrp/cvxpylayers/pull/157

### CODE REVIEW

**Comment by PTNobel** [comment]
Awesome. I'll probably hijack this PR to make the needed CI changes too

File: diffcp/__init__.py [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 from diffcp.cone_program import solve_and_derivative, \
     solve_and_derivative_batch, \
-    solve_and_derivative_internal, SolverError
+    solve_and_derivative_internal, SolverError, \
+    solve_only_batch, solve_only, \
+    solve_internal
```
Comment (PTNobel) [inline diffcp/__init__.py:7]: `SolverError` should still be last

Comment (healeyq3) [reply diffcp/__init__.py]: My bad - I just pushed that change. Hopefully that's right now.
Also, for learning purposes, is putting `SolverError` last in an `__init__.py` file generally a best practice, or is it a cvxgrp stylistic preference, *i.e.*, in a statement like `from diffcp.cone_program import` ..., is it best to put functions and then classes?

Comment (PTNobel) [reply diffcp/__init__.py]: It's just a cvxgrp preference. I think Ruff and some other tools have opinions about how imports should be organized.

Comment (healeyq3) [reply diffcp/__init__.py]: Good to know, thanks!

**Review by PTNobel — APPROVED** [review]

**Pull request merged**